src/components/TransactionConfirmationModal/index.tsx

```diff
@@ -1,5 +1,5 @@
 import { Trans } from '@lingui/macro'
-import { Currency } from '@uniswap/sdk-core'
+import { Currency, SupportedChainId } from '@uniswap/sdk-core'
```

Comment:
Should `SupportedChainId` come from `constants/chains`?

Reply:
Could we have a follow-up PR where we migrate to that SDK constants? I had a brief look and I am not sure whether it has 100% overlap with our constants, but this could help.